swaybar/render.c

```diff
@@ -165,11 +163,11 @@ static enum hotspot_event_handling block_hotspot_callback(
 	struct i3bar_block *block = data;
 	struct status_line *status = output->bar->status;
 	return i3bar_block_send_click(status, block, x, y,
-			x - (double)hotspot->x / output->scale,
-			y - (double)hotspot->y / output->scale,
-			(double)hotspot->width / output->scale,
-			(double)hotspot->height / output->scale,
-			output->scale, button);
+			x - (double)hotspot->x,
+			y - (double)hotspot->y,
+			(double)hotspot->width,
+			(double)hotspot->height,
+			1, button);
```

Comment:
Hm, not sure this part about input event handling is quite right. `process_hotspots` multiplies incoming pointer coordinates by the scale. If hotspot coords are now unscaled, maybe we need to remove that multiplication?

Also I think we should keep sending `output->scale` as the second-to-last parameter.

Reply:
Thanks for pointing out the input code. I completely missed testing mouse events/buttons also (touch points untested but should work fine). Addressed in latest commit.